mtp_noms_ops/apps/security/forms/check.py

```diff
@@ -198,18 +203,30 @@ class AcceptOrRejectCheckForm(GARequestErrorReportingMixin, forms.Form):
         required=False,
         label=CHECK_DETAIL_FORM_MAPPING['decision_reason'],
     )
-    fiu_investigation_id = forms.CharField(
+    has_fiu_investigation_id = forms.BooleanField(
```

Comment:
i did revert the validation of the subfields. there's a separate branch for that. however we do need these to feed into the templates. otherwise we'll have to have a separate template for each of the 4 checkboxes… that's surely worse to keep in sync..?